### ISSUE TITLE
twshoot doesn't trim the image.

### ISSUE THREAD

**Issue by TanukiMa** [travisbrown/cancel-culture#58]
I am grateful to the developers.

By the way, twshoot doesn't trim the image.

Why?

```
$ ~/cancel-culture$ ./target/release/twshoot 1503631923154984960
$  $ file 1503631923154984960*.png
1503631923154984960-full.png: PNG image data, 800 x 4000, 8-bit/color RGBA, non-interlaced
1503631923154984960.png:      PNG image data, 596 x 3307, 8-bit/color RGBA, non-interlaced
```

![1503631923154984960](https://user-images.githubusercontent.com/39878909/158337644-4c2a5e56-ba7c-4b11-ae69-517e77ab0187.png)
![1503631923154984960-full](https://user-images.githubusercontent.com/39878909/158337617-3548f251-2f5e-486e-84c7-d20bc7a23cae.png)

**Comment by travisbrown** [comment]
Thanks for the report! Twitter has made some changes to the rendering that have broken the heuristics for finding the borders for the screenshot. I've been meaning to fix this but haven't gotten around to it. I'll take a look today.

**Comment by travisbrown** [comment]
@TanukiMa Mind trying [this branch](https://github.com/travisbrown/cancel-culture/tree/topic/twshoot-fix), from #59? It's not a perfect solution but it should work.

**Comment by travisbrown** [comment]
Okay, I was able to reproduce the error and think it's fixed in the latest commit on that branch. Thanks for the detailed info.

**Comment by travisbrown** [comment]
Fixed (probably) in #59. If not please reopen.
